app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -167,6 +168,7 @@ object AppRepository {
             executionContext: ExecutionContext = AppExecutionContext,
             databaseScope: DatabaseScope = DatabaseScope.of(executionContext, AppExceptionHandler(logging)),
             networkScope: NetworkScope = NetworkScope.of(executionContext, AppExceptionHandler(logging)),
+            okHttpClient: OkHttpClient = CustomHttpClient.createHttpClient(),
```

Comment:
Since I cannot comment directly on the method head - who calls this method or is that a pure test concern? Was not able to find any invocation directly.